src/test/resources/elasticsearchdata/john-smith.json

```diff
@@ -78,5 +78,11 @@
     "officer": {"code": "NO2AAMU", "forenames": "Unallocated", "surname": "Staff"},
     "probationArea": {"code": "NO2", "description": "NPS London"},
     "notes": "Level 1 Cat 2 notes"
+  },
+  "probationStatus": {
```

Comment:
I was just looking at the ConvictionService in community-api and it looks like you can get null values for termination date and in breach... I wonder if we could do with some tests around those null values?  (In another PR if you think it is necessary)

Reply:
Fair point.